index.html

```diff
@@ -56,14 +56,49 @@
     </style>
     <script src="https://ajax.googleapis.com/ajax/libs/jquery/2.1.3/jquery.min.js"></script>
     <script>
-      // Put some code in here!
+    var intervalID;
+    var running = false
+    $(document).ready(function() {
+      var value1 = $("#counter").val();
```

Comment:
Running `$("")` is actually pretty computationally expensive. I'd recommend running it once at the top of your script and saving it to a variable, and then using that variable going forward.